src/game/scenes/lobby.c

```diff
@@ -641,10 +641,10 @@ void lobby_entered_name(component *c, void *userdata) {
             event.peer->data = nat;
             strncpy_or_truncate(local->name, textinput_value(c), sizeof(local->name));
 
-            char version[15];
+            char version[30];
             // TODO support git version when not on a tag
-            snprintf(version, 14, "%d.%d.%d", V_MAJOR, V_MINOR, V_PATCH);
-            version[14] = 0;
+            snprintf(version, 29, "%d.%d.%d%s", V_MAJOR, V_MINOR, V_PATCH, V_LABEL);
+            version[29] = 0;
```

Comment:
Using `snprintf(version, sizeof(version), ..` would make the `version[29] = 0` redundant here and generally sizeof is nicer to use than magic constants.